index.js

```diff
@@ -91,17 +91,25 @@ const globDirs = (task, fn) => {
 	return fn(task.pattern, options);
 };
 
-const getPattern = (task, fn) => task.options.expandDirectories ? globDirs(task, fn) : [task.pattern];
+const getIgnoreFiles = options => options && options.gitignore ? '**/.gitignore' : options && options.ignoreFiles;
+
+const getPattern = (task, fn) =>
+	task.options.expandDirectories ? globDirs(task, fn) : [task.pattern];
 
 const getFilterSync = options => {
-	return options && options.gitignore ?
-		gitignore.sync({cwd: options.cwd, ignore: options.ignore}) :
+	const ignoreFiles = getIgnoreFiles(options);
+	return ignoreFiles ?
+		ignore.sync(ignoreFiles, {cwd: options.cwd, ignore: options.ignore}) :
 		DEFAULT_FILTER;
 };
 
 const globToTask = task => glob => {
 	const {options} = task;
-	if (options.ignore && Array.isArray(options.ignore) && options.expandDirectories) {
+	if (
+		options.ignore &&
+		Array.isArray(options.ignore) &&
+		options.expandDirectories
+	) {
```

Comment:
Please don't do unrelated changes. Undo all unrelated changes.

Reply:
Undone. It's the dang autoformatter.